deploy/helm/rocketride/templates/postgres-statefulset.yaml

```
@@ -0,0 +1,113 @@
+    app.kubernetes.io/component: postgres
+spec:
+  serviceName: {{ include "rocketride.fullname" . }}-postgres
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "rocketride.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: postgres
+  template:
+    metadata:
+      labels:
+        {{- include "rocketride.selectorLabels" . | nindent 8 }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)